### DISCUSSION TITLE
Why `Box` have `sx` prop?

### DISCUSSION THREAD

**Discussion by PeterlitsZo** [system-ui/theme-ui#2126] [General] [answered]
I find this in `Box.js`:

```
// ...
const sx = (props) => css(props.sx)(props.theme)
// ...

export const Box = styled('div', {
  shouldForwardProp,
})(
  {
    boxSizing: 'border-box',
    margin: 0,
    minWidth: 0,
  },
  base,
  variant,
  space,
  color,
  sx,
  (props) => props.css
)
```

The question is: The jsx funtion that theme has, it will remove the sx prop... so not `sx` prop will be saved for `Box`. so... why it has `sx` prop? Is it a bug?

**Comment by hasparus** [comment, marked as answer]
Hey, firstly thank you for the attention and your activity in this repo. I really appreciate it.

So, this is not a bug.
Box supports both `sx` and `className`. You can use the `Box` component without configuring JSX Pragma, without using `jsx` function.

**Reply by PeterlitsZo** [reply]
Thanks, I get it!
